vm-setup/roles/libvirt/templates/baremetalvm.xml.j2

```diff
@@ -1,11 +1,18 @@
-<domain type='{{ libvirt_domain_type }}'>
+{% set is_arm = flavors[item.flavor].architecture == 'aarch64' %}
+<domain type='{% if is_arm %}qemu{% else %}{{ libvirt_domain_type }}{% endif %}'>
```

Comment:
I think a bit more correct would be to use `qemu` whenever `flavors[item.flavor].architecture != libvirt_arch` (not sure we have any chances of running metal3-dev-env on arm, but who knows..)

Reply:
Done